doc/api/modules.md

```diff
@@ -481,7 +481,7 @@ folders as modules, and work for both `require` and `import`.
 
 If the module identifier passed to `require()` is not a
 [core](#core-modules) module, and does not begin with `'/'`, `'../'`, or
-`'./'`, then Node.js starts at the parent directory of the current module, and
+`'./'`, then Node.js starts at the current directory of the current module, and
```

Comment:
@aduh95 is that true? From my experience a file is in a current directory. It is not a parent directory.  But I think omitting current entirely also encapsulates this, as the word maybe does not add value. So I added the suggestion.

Reply:
Well a directory has "children", so from the point of view of the file, it is inside a "parent" directory. I think it's a pretty common analogy to use for a file system, but maybe it could mean something else in this context? What would be a "current" directory? You say it is "not a parent directory", what do you mean by that?
Asking those questions because I'm trying to understand your original point, English is not my primary language so I prefer to avoid any ambiguity. To be clear the current version looks OK to me.

Reply:
@aduh95 that is not my interpretation of current/parent directory. If I do `ls -a` the `.` significes current directory and `..` signifies parent directory. The same as you get when you do `pwd` you get the [current working directory](https://en.wikipedia.org/wiki/Pwd). If there is no "current working directory", you are basically saying a file never has a "current working directory"?

Have I always been wrong in thinking a file´s parent directory is not the one it currently resides in, but the one one-level above? 😮 

To be clear, I am _also_ find with the current solution we have now haha.

Reply:
A parent directory of the current location is “..”, and the current directory for it is “.”.

The directory containing a file is not the file’s parent directory, it’s the file’s directory.

Reply:
> `.` significes current directory and `..` signifies parent directory

`..` is the parent directory of `.`, that I agree. I think it's weird to apply a different logic to files and directory: the parent directory of `./a/b/` is `./a/`, but the parent directory of `./a/b` would be `./`?


> The same as you get when you do `pwd` you get the [current working directory](https://en.wikipedia.org/wiki/Pwd).

The cwd is the context in which the _program_ is executed from, you can't really use that for a file I don't think.

Reply:
@aduh95 then a file can never have a current directory by that logic?

Reply:
The `./a/b` file is *in* (contained within) `./a`, but the file's parent directory is absolutely `./`.

I think a file doesn't really have a current or a parent directory, because those terms are generally only used relative to a current working directory - meaning, when discussing a file, the parent directory is relative to `dirname()` of the file, not the file itself.

Reply:
https://linux.die.net/man/1/dirname

> Print NAME with its trailing /component removed; if NAME contains no /'s, output '.' (meaning the current directory).

Reply:
> @aduh95 then a file can never have a current directory by that logic?

No in my logic files never have a "current" directory, or at least I never though of it this way.

> I think a file doesn't really have a current or a parent directory, because those terms are generally only used relative to a current working directory

This ☝️ 

> https://linux.die.net/man/1/dirname
> 
> > Print NAME with its trailing /component removed; if NAME contains no /'s, output '.' (meaning the current directory).

In that context, `.` is the cwd, so it makes sense to use the term "current" (even to me and my _special_ logic 😅). FWIW `dirname` does not differentiate files and directories:

```console
$ dirname ./a/b/
./a
$ dirname ./a/b 
./a
```

Reply:
We're good at bikeshedding :D But I enjoy the discussion.

Consider it like this: Directory stems from the word "folder" in the real world. A folder has "files" in it.
"Please take the file from the current folder and put it in the green folder" is a perfectly legal English sentence.

Ergo: To me a file has a current directory.

I think the examples `dirname` (and you provied) by my logic proves it has a "current" directory 😄

Reply:
btw @aduh95 feel free to merge when you get a chance